tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -39,6 +39,10 @@ SDW_DMIC_STREAM=SDW0-Capture"
 SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
 SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
 SDW_JACK_IN_STREAM=Capture-SimpleJack"
+
+# Jack codec + SmartAmp topology. No SDW_DMIC connection
```

Comment:
@bardliao @plbossart Do we have some convention to expose dmic/no-dmic in the tplg filename. I guess now we have not had any conflicts where we'd need the same configuration with and without dmic, so I guess this is ok, but how will this scale? Add "-dmic" when/if such cames arises?

Reply:
if there is no rt715 in the name then there's no dmic IIRC. @bardliao can you confirm?

Reply:
> if there is no rt715 in the name then there's no dmic IIRC. @bardliao can you confirm?

Right, but I guess what @kv2019i mean is no PCH DMIC. I.e. sof-mtl-rt713-l0-rt1316-l12 with or without PCH DMIC.